config/rbac/daemon_role.yaml

```diff
@@ -38,6 +38,7 @@ rules:
   - get
   - list
   - watch
+  - patch
```

Comment:
it is recommended to avoid assign object patch privilege to daemons, so the container id of killed container can be saved in crr status

Reply:
done